pkg/speaker/vip/keepalived.go

```diff
@@ -0,0 +1,193 @@
+package vip
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-logr/logr"
+	"github.com/openelb/openelb/pkg/constant"
+	"github.com/openelb/openelb/pkg/speaker"
+	"github.com/openelb/openelb/pkg/util"
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"strings"
+)
+
+type KeepAlive struct {
```

Comment:
`KeepAlive` -> `KeepAlived`,  TCP have the `KeepAlive`, and we are talk about the software of `KeepAlived`,right?

Reply:
ok